charts/gateway/production-values.yaml

```diff
@@ -423,9 +446,12 @@ config:
       masterSet: mymaster
       # If the subChart is not enabled sentinel nodes need to be set
       nodes:
-      - <host>:<port>
-      - <host>:<port>
-      - <host>:<port>
+      - host: <host>
```

Comment:
[_helpers.tpl](https://github.com/CAAPIM/apim-charts/pull/324/files#diff-d2d76d21ba43b4dbe74bd202897af619c24c47ef68f354d7dc1bb5cb0ead21cc) need to be updated for this change accordingly.

Reply:
gateway.redisSentinelNodes removed